internal/cache/metadata/stat_cache.go

```diff
@@ -74,6 +73,10 @@ type entry struct {
 	expiration time.Time
 }
 
+func (e entry) Size() uint64 {
+	return uint64(unsafe.Sizeof(gcs.Object{}) + unsafe.Sizeof(entry{}))
```

Comment:
Just curious, what's the size of entry right now?

Reply:
unsafe.Sizeof(gcs.Object{}) = 296
unsafe.Sizeof(entry{}) = 32

Reply:
The actual size per cache size will be probably more because of other overheads like copy of key, pointers, map structure etc.

Reply:
In that case, should we keep some overhead size just to capture correct upper limit?

We can keep a TODO to revisit again to add the overhead limit.

Reply:
@raj-prince parked the question and another related question in https://github.com/GoogleCloudPlatform/gcsfuse/discussions/1526  for now.